spec/takeaway_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'takeaway'
+require 'menu'
+require 'order'
+
+describe Takeaway do
+  let(:menu) { double(:menu, print: printed_menu) }
+  let(:printed_menu) { {
+    "Chicken bits" => 5,
+    "Burger" => 5,
+    "Pizza" => 8,
+    "Chips" => 3
+  } }
+  let(:order) { Order.new }
```

Comment:
Could this be updated to use a London style test, where the Order object is doubled.